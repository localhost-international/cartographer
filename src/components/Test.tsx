import React from 'react'
import styled from 'styled-components/native'

export default function Test() {
  return (
    <TestView>
      <Text>Testing, testing, 1, 2, 3</Text>
    </TestView>
  )
}

const TestView = styled.View`
  background-color: rgba(255, 0, 0, 0.25);
`
const Text = styled.Text`
  color: red;
`
