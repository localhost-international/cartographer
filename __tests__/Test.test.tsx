import React from 'react';
import renderer from 'react-test-renderer';

import Test from '../src/components/Test';

import 'jest-enzyme';
import 'jest-styled-components';

it('Test component renders', () => {
  const component = <Test />;
  const tree = renderer.create(component).toJSON();
  expect(tree).toMatchSnapshot();
});
