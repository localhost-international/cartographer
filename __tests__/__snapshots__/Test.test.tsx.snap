// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component renders 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "red",
      },
    ]
  }
>
  Testing, testing, 1, 2, 3
</Text>
`;
